File: Majo/Act-Instructor_Henry/lenguajes/ruby/mayo/31-05-24.rb
puts "Número de estudiantes:"
a = gets.chomp.to_f

for i in 1..a
  puts "Nombre del estudiante:"
  j = gets.chomp

  puts "Las notas van de 1 a 10"

  puts "La nota 1 es:"
  n1 = gets.chomp.to_i

  puts "La nota 2 es:"
  n2 = gets.chomp.to_i

  puts "La nota 3 es:"
  n3 = gets.chomp.to_i

  sum = n1 + n2 + n3
  prom = sum / 3

  if prom > 6
    puts "El estudiante #{j} sí aprobo la materia"
  else
    puts "El estudiante #{j} no aprobo la materia"
  end
end

File: Majo/Act-Instructor_Henry/lenguajes/dart/mayo/31-05-24.dart
import 'dart:io';

void main() {
  print("Número de estudiantes:");
  int a = int.parse(stdin.readLineSync()!);

  for (var i = 1; i <= a; i++) {
    print('Nombre del estudiante:');
    String? j = stdin.readLineSync();

    print('Las notas van de 1 a 10');

    print('La nota 1 es:');
    int n1 = int.parse(stdin.readLineSync()!);

    print('La nota 2 es:');
    int n2 = int.parse(stdin.readLineSync()!);

    print('La nota 3 es:');
    int n3 = int.parse(stdin.readLineSync()!);

    int sum = n1 + n2 + n3;
    double prom = sum / 3;
  
    if (prom > 6) {
      print('El estudiante $j sí aprobo la materia');
    } else {
      print('El estudiante $j no aprobo la materia');
    }
  
  }
}